torch/_dynamo/exc.py

```diff
@@ -271,6 +271,10 @@ class FailOnRecompileLimitHit(Exception):
     pass
 
 
+class PackageError(Exception):
```

Comment:
Should this subclass Exception? NotImplementedException? RuntimeException?

Reply:
Maybe just TorchDynamoException makes sense here